docs/pages/versions/unversioned/sdk/audio.mdx

```diff
@@ -14,6 +14,8 @@ import { PlatformTags } from '~/ui/components/Tag';
 
 `Audio` from `expo-av` allows you to implement audio playback and recording in your app.
 
+> **Info** Audio recording APIs are not available on tvOS (Apple TV).
```

Comment:
```suggestion
```

i'd prefer for this to just be annotated alongside the related classes/methods

Reply:
@brentvatne I just added a commit to do this -- there are MANY comments that need to be added. To really do this properly so that it scales, we need to support a doc tag like `@!tvos` ("Not for tvOS"), that way we don't have to add so many tags.

Reply:
I'm inclined to have both the "Info" comment in the summary page and the class/method annotations...

Reply:
the problem with the info is that, imo, it's somehow both 1) easy to miss and 2) not the most relevant information for the vast majority of people at the time that they are reading that section of the page. you really want to know this on the class/function level.

> we need to support a doc tag like @platform !tvos ("Not for tvOS")

that's a really good point! what do you think @amandeepmittal @Simek?

Reply:
> > we need to support a doc tag like https://github.com/platform !tvos ("Not for tvOS")

> that's a really good point! what do you think @amandeepmittal @Simek?

We have decide in the past that we should add platform labels to every supported platform instead of negation (non-supported). This is a long term goal, which started the effort of improvement for platform tags in the first place. Basically, we want to mimic a bit more Apple docs approach:
* https://developer.apple.com/documentation/coremotion/cmmagnetometerdata

Adding missing `@platform android` or `@platform ios` tag is exactly what I have in mind when talking with Doug about marking tvOS compatibility better (so marking the unsupported methods in API which globally mark tvOS as supported), which lead to creation for the referenced Linear task.